prow/deck/pdb.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: deck
+spec:
+  minAvailable: 90%
```

Comment:
I prefer `maxUnavailable: 1`. I think with the current PDB you'd actually block all evictions as removing even 1 of 2 pods would result in the budget being disrupted